addons/mail/static/src/core/common/record.js

```diff
@@ -125,7 +148,13 @@ export class RecordList extends Array {
             },
         });
     }
-
+    /**
+     * @param {R|any} val
+     * @param {(R) => void} fn
+     */
+    _preinsert(val, fn) {
+        preinsert(val, this.owner, this.name, fn);
+    }
```

Comment:
looks like not very useful indirection

Reply:
It's to avoid to repeat `this.owner` and `this.name` for the 5 occurrences in this file (`push`, `unshift`, `add`, `delete`, `[Record.many][index] =`).
There are 2 occurrences in store_service that needs the preinsert (`[Record.many]=`  and `[Record.one]=`)